src/rest-interfaces/index.ts

```diff
@@ -63,3 +63,89 @@ export type ErrorCode =
 	| "SSO_LOGIN"
 	| "RESOURCE_NOT_FOUND"
 	| "UNKNOWN";
+
```

Comment:
Same indentation issue here

Reply:
Same here, will be done in [ARC-2467](https://softwareteams.atlassian.net/browse/ARC-2467)

Reply:
Done